### PR TITLE
compatibility docs: fixup steering torque star

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: selfdrive/car/tests/test_docs.py [hunk 1]
```diff
@@ -39,9 +39,18 @@ def test_naming_conventions(self):
...
   def test_torque_star(self):
+    failures = []
     for car in self.all_cars:
       if car.car_name == "honda":
-        self.assertTrue(car.row[Column.STEERING_TORQUE] in (Star.EMPTY, Star.HALF), f"{car.name} has full torque star")
+        if car.row[Column.STEERING_TORQUE] not in (Star.EMPTY, Star.HALF):
+          failures.append(f"{car.name} has full torque star")
+      elif car.car_name in ("toyota", "hyundai"):
+        if car.row[Column.STEERING_TORQUE] == Star.EMPTY:
+          failures.append(f"{car.name} has no torque star")
```
Comment (adeebshihadeh) [inline selfdrive/car/tests/test_docs.py:49]: use unittest's subTest for this

File: selfdrive/car/subaru/values.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ class SubaruCarInfo(CarInfo):
 
 
 CAR_INFO: Dict[str, Union[SubaruCarInfo, List[SubaruCarInfo]]] = {
-  CAR.ASCENT: SubaruCarInfo("Subaru Ascent 2019-20", "All"),
+  CAR.ASCENT: SubaruCarInfo("Subaru Ascent 2019-20", "All", good_torque=True),
   CAR.IMPREZA: [
```
Comment (adeebshihadeh) [inline selfdrive/car/subaru/values.py:45]: no half star for the impreza?

Comment (sshane) [reply selfdrive/car/subaru/values.py:45]: Impreza has `0.8234374840709592` of measured accel in the old and new data

Comment (sshane) [reply selfdrive/car/subaru/values.py:45]: Looking at some segments, it's hitting around 1.5 with no steering pressed

**Pull request merged**